### ISSUE TITLE
Zink & Dozen issues found using 22.2rc1..

### ISSUE THREAD

**Issue by oscarbg** [pal1000/mesa-dist-win#95]
Hi,

I just want to comment in case you replicate also that Zink & Dozen have issues running simple apps on Win..
Dozen is new so no problem.. 
but Zink is more serious as Mesa 22.0.x are OK..
I built myself Mesa and I already opened a bug days before your release:
https://gitlab.freedesktop.org/mesa/mesa/-/issues/6331
but for Zink not yet, just opened now:
https://gitlab.freedesktop.org/mesa/mesa/-/issues/6337

**Comment by pal1000** [comment]
Tested dozen with vkcube and it crashed.

**Comment by pal1000** [comment]
@oscarbg, where did you get wglgears from?

**Comment by oscarbg** [comment]
I believe from http://www2.cs.uidaho.edu/~jeffery/win32/

**Comment by pal1000** [comment]
I thought it was from somewhere else, but OK. I tested and reproduced the problem with zink. I also tested 64-bit PPSSPPP and it didn't even manage to initialize graphics acting like there is no OpenGL driver.

**Comment by Matthaiks** [comment]
22.1.0-rc3 and Zink

Quake: black screen
Quake 2: won't launch

22.0.2 is OK.

**Comment by oscarbg** [comment]
@pal1000 vkcube bug on dozen is fixed.. but zink still is broken.. wglgears still doesn't render correctly..

**Comment by pal1000** [comment]
Report upstream, please.

**Comment by Matthaiks** [comment]
22.1.0

Zink is still broken.

**Comment by oscarbg** [comment]
@Matthaiks what GPU? 
My own build of 22.1 works on Nvidia GPU.. but seems I still have issues running Zink on AMD GPU..

**Comment by Matthaiks** [comment]
@oscarbg AMD Radeon RX Vega 8

**Comment by oscarbg** [comment]
yep still broken on my Vega also.. also tested on Win on Intel Skylake iGPU and also broken.. so seems only works on Nvidia on Windows..

**Comment by oscarbg** [comment]
for Intel on Windows maybe still this bug is present:
https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/11427
